### PR TITLE
Checking if file in cache is stale

### PR DESCRIPTION
**Pull request by stulees**
- Database now includes timestamp data, which will add `createdAt` and `updatedAt` values
- File Controller has `getTimestampData` function for retrieving timestamp data from saved header
- File Controller has `isStale` function for determining if file in cache was last checked over 20 minutes ago
- Unit tests added

### CODE REVIEW

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Can you review this without test coverage first? I'm unsure if I'm doing the right thing here - https://github.com/Rise-Vision/rise-cache-v2/blob/feature/check-new-file/app/controllers/file.js#L47-L48 . What happens is the browser doesn't display the image anymore as I guess it's unable to read the stream (?). I think this is a problem because we're still writing and reading to the same path (in download folder). Once there is a change to the file, then the browser is displaying the image again. 

What do you recommend doing?

**Comment by donnapep** [comment]
@stulees I've actually changed this logic quite a bit in my current PR. I wonder if the issue you're experiencing might be resolved by my changes?

**Comment by stulees** [comment]
@donnapep I'm not sure, maybe. Looks like I will need to do a substantial rebase nonetheless if we merge yours first.

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Can you review again, I've fixed the problem. Test coverage still to come. Thanks

**Comment by stulees** [comment]
@rodrigopavezi As we discussed, I've left the actual downloading of the file to do on a future card. At the moment it just logs that the file is stale. I tried but couldn't see a way to write test coverage. Can you please review, thanks.

**Comment by rodrigopavezi** [comment]
@stulees I think you can move the isStale function into the file controller and reuse the method getHeaders from my PR. There are some example of test coverage for that there as well.  cheers

**Comment by stulees** [comment]
@donnapep @rodrigopavezi Please review again, thanks.

File: app/controllers/file.js [hunk 1]
```diff
@@ -13,6 +12,7 @@ const FileController = function(url, header) {
   this.header = header;
   this.fileName = fileSystem.getFileName(this.url);
   this.path = fileSystem.getPath(this.url);
+  this.updatedDuration = 1200000;
```
Comment (rodrigopavezi) [inline app/controllers/file.js]: @stulees wouldn't be better to have this property on the config file? cheers

**Comment by rodrigopavezi** [comment]
@stulees just a small thing otherwise looks good 👍 cheers

**Comment by stulees** [comment]
@rodrigopavezi Please review last commit, thanks.

**Comment by rodrigopavezi** [comment]
@stulees looks good 👍 cheers

**Pull request merged**